src/modules/sales/sales.service.ts

```
@@ -0,0 +1,383 @@
+      order: { id: 'DESC' },
+    });
+
+    let sequence = 1;
+    const datePrefix = `${year}${monthStr}${dayStr}`;
+
+    if (lastSale?.saleNumber) {
+      const parts = lastSale.saleNumber.split('-');
+      if (parts.length === 3 && parts[1] === datePrefix) {
+        sequence = parseInt(parts[2], 10) + 1;
+      }
+    }
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)